en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md

```diff
@@ -61,7 +61,7 @@ class ProductAdmin extends ModelAdmin
 ```
 
 In order to localize the menu title in different languages, use the
-`<classname>.MENUTITLE` entity name, which is automatically created when running
+`<classname>.MENUTITLE_CMS6` entity name, which is automatically created when running
```

Comment:
Note that if we do go ahead with this change, I'll also need to add an entry to the 6.0 changelog so people can update their localisation keys in their projects for their custom admin sections.